test/Directory.Build.props

```diff
@@ -16,4 +16,9 @@
     -->
   </PropertyGroup>
 
+  <ItemGroup>
+    <Using Include="Altinn.App.Common.Tests" />
+    <Using Include="Altinn.App.Common.Tests.Auth" />
+  </ItemGroup>
```

Comment:
What makes these namespaces so different from every other namespace?

Reply:
They are used a lot across both projects, so figured we would save some usings